src/amo/api/users.js

```diff
@@ -89,3 +89,18 @@ export function deleteUserPicture({ api, userId }: {|
     state: api,
   });
 }
+
+export function deleteUserAccount({ api, userId }: {|
+  api: ApiStateType,
+  userId: UserId,
+|}) {
```

Comment:
I just noticed that we don't have any return types for any of these functions, not even the ones that are expected to return something, and I think that's something we should fix. Could be a follow-up to this as it's unrelated to this patch.

Reply:
Do we have a type for that? or is it just a generic `Promise`? I'll file an issue.

Reply:
In the collections API we use `Promise<void>` if we do not expect a return value, and `Promise<type>` if we do, e.g., `Promise<ExternalCollectionDetail>`.

Reply:
See: https://github.com/mozilla/addons-frontend/issues/5177